petclinic-frontend/src/shared/models/path.routes.ts

```diff
@@ -1,6 +1,7 @@
 export enum AppRoutePaths {
   Default = '/',
   Inventories = '/inventories',
+  Vet = '/vet', //added vet's route path
```

Comment:
Comment code like this should not be left in PRs.

Reply:
The comments have been removed.